libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -259,6 +262,16 @@ export class PageComponent
     this.windowRef.nativeWindow.addEventListener(selectedTabClickEvent, () => {
       this.content.scrollToTop(KirbyAnimation.Duration.LONG);
     });
+
+    // Intercept back-button click events, defaulting to the built-in click-handler.
+    if (this.backButtonClicked.length == 0) {
+      this.backButtonClicked
+        .pipe(takeUntil(this.ngOnDestroy$))
+        .subscribe(this.backButtonDelegate.onClick.bind(this.backButtonDelegate));
+    }
+    this.backButtonDelegate.onClick = (event: Event) => {
+      this.backButtonClicked.emit(event);
+    };
```

Comment:
We could consider separating all of this into a `InterceptBackButtonClicks` method to make it more clear that this code belongs together and what its responsibility is

Reply:
Done